gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -14352,6 +14353,12 @@ private long getEntryOverhead(RegionEntry entry){
     Object key = entry.getRawKey();
     if (key != null) {
       entryOverhead += ReflectionSingleObjectSizer.INSTANCE.sizeof(key);
+    }else{
+      // first key.
+      Object firstKey = this.getRegionMap().keySet().iterator().next();
+      if(firstKey != null){
+        entryOverhead += ReflectionSingleObjectSizer.INSTANCE.sizeof(firstKey);
+      }
```

Comment:
I am a bit confused here. CachedDeserializableFactory.calcMemSize() will evaluate size of rawKey and and not the RegionEntry object overhead.  Do you suggest to use calcMemSize only for rawKey ?